packages/stream_chat_flutter/lib/src/thread_header.dart

```diff
@@ -179,4 +191,21 @@ class ThreadHeader extends StatelessWidget implements PreferredSizeWidget {
 
   @override
   final Size preferredSize;
+
+  bool _getEffectiveCenterTitle(ThemeData theme) {
```

Comment:
Since we are using the same function in `ChannelHeader` as well as `ThreadHeader` do you think it would be a good idea to make it a global helper function and use that in both places, instead of duplicating the code?